src/test/java/com/rultor/agents/github/ReportsTest.java

```diff
@@ -56,9 +68,52 @@
     public void reportsRequestResult() throws Exception {
         final Repo repo = new MkGithub().randomRepo();
         final Issue issue = repo.issues().create("", "");
+        final Talk talk = ReportsTest.example(repo, issue);
+        final Agent agent = new Reports(repo.github());
+        agent.execute(talk);
+        MatcherAssert.assertThat(
+            talk.read(),
+            XhtmlMatchers.hasXPath(XPATH)
+        );
+    }
+
+    /**
+     * Reports can report a result of a request, when stop command fails.
+     * @throws Exception In case of error.
+     */
+    @Test
+    public void reportsRequestResultWhenStopFails() throws Exception {
+        final Repo repo = new MkGithub().randomRepo();
+        final Issue issue = repo.issues().create("Bug", "stop it please");
+        final Talk talk = ReportsTest.example(repo, issue);
         final Agent agent = new Reports(repo.github());
-        final Talk talk = new Talk.InFile();
-        talk.modify(
+        agent.execute(talk);
+        MatcherAssert.assertThat(
+            talk.read(),
+            XhtmlMatchers.hasXPath(XPATH)
+        );
+        MatcherAssert.assertThat(
+            "Comment contains warning about stop request",
+            repo.issues().get(1).comments().get(1).toString().contains(
+                ReportsTest.PHRASES.getString("Reports.stop-fails").replace(
+                    "%s",
+                    ""
+                )
+            )
+        );
+    }
+
+    /**
+     * Create Talk, that will be used to test Reports.
+     * @param repo Repository
+     * @param issue Issue
+     * @return Example of Talk
+     * @throws IOException In case of error.
```

Comment:
@xupyprmv dot at the end of this line is not needed, it will be soon prohibited by Qulice